### PR TITLE
Delete Playlist Record by ID

### LINKED ISSUE #32: Delete Playlist Object - Happy Path
DELETE /api/v1/playlists/:id

This route will delete the playlist that matches the ID passed in. It returns a 204 status code with a message that the record has been deleted.

### LINKED ISSUE #33: Delete Playlist Object - Sad Path
DELETE /api/v1/playlists/:id

If the playlist record can't be found it returns a 404 status code and error message stating:

error: "Playlist not found".

### PR DESCRIPTION
**Pull request by cjkelling**
This closes #32 closes #33 

Test: Add tests to verify the deleting of a playlist record from the database by its ID. Tests sad path if a record cannot be found. 

Route: /api/v1/playlists/:id  DELETE

### CODE REVIEW

File: routes/api/v1/playlists.js [hunk 1]
```diff
@@ -27,4 +27,14 @@ router.post('/', (req, res) => {
...
+router.delete('/:id', (req, res) => {
+  database('playlists').del().where({id: req.params.id}).then(playlist => {
+    if (playlist) {
+      res.status(204).send();
+    } else {
+      res.status(404).json({error: "Playlist not found"});
+    }
+  }).catch(err => res.status(404).json({error: err}))
+});
+
```
Comment (Not-Zorro) [inline routes/api/v1/playlists.js]: Good handling edge cases

File: tests/delete_playlist.spec.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    it('Should respond with a 404 if fav not found in db', async () => {
+      const res = await request(app)
+        .delete('/api/v1/playlists/1');
+
+      expect(res.statusCode).toBe(404);
+      expect(res.body).toEqual({error: "Playlist not found"});
+    })
+  })
+})
```
Comment (Not-Zorro) [inline tests/delete_playlist.spec.js]: Greate test for happy/sad paths
